src/error/views/ErrorView.tsx

```diff
@@ -41,6 +51,12 @@ const ErrorView: FunctionComponent<ErrorViewProps> = ({
 			<Container size="medium" mode="horizontal">
 				<Blankslate body="" icon={errorIcon} title={errorMessage}>
 					{children}
+					{actionButtons.includes('home') && (
```

Comment:
Als het zo geïmplementeerd wordt mss best gwn een prop `showBackToHomeButton` o.i.d. gebruiken

Reply:
er zullen mogelijks nog andere redirect knoppen toegevoegd worden
bv: 
* ga terug naar home maar voor het dashboard
* of als de detail van een collectie faalt dan kan je naar collectie overview geredirect worden

Reply:
Ok, maar je gaat er toch nooit meer als 1 tonen dan of wel? Als dat wel het geval is dan is het goed zo.

Reply:
ik dacht mogelijks wel.
eg: terug naar home, rapporteer een probleem